app/code/Magento/Directory/Model/PriceCurrency.php

```diff
@@ -148,13 +146,18 @@ protected function getStore($scope = null)
     }
 
     /**
-     * Round price
-     *
-     * @param float $price
-     * @return float
+     * @inheritdoc
      */
     public function round($price)
     {
         return round($price, 2);
     }
+
+    /**
+     * @inheritdoc
```

Comment:
As there is no parent method declaration anymore, please annotate a short description, params and return here.